docs/scos/dev/feature-walkthroughs/202204.0/click-and-collect-feature-walkthrough/product-offer-service-point-availability-subdomain-walkthrough.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Click and Collect feature Product Offer Service Point Availability subdomain walkthrough
+last_updated: Nov 02, 2023
+description: TODO
+template: concept-topic-template
+---
+
+# Product Offer Service Point Availability
+
+Refers to the availability calculation.
+
+[Install the Product Offer Service Points Availability feature](/docs/pbc/all/install-features/{{page.version}}/install-the-product-offer-service-points-availability-feature.html)
+
+## 1. Modules:
+
+| MODULE                                                         | EXPECTED DIRECTORY                                                                   |
+|----------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| ProductOfferServicePointAvailabilitiesRestApi                  | vendor/spryker/product-offer-service-point-availabilities-rest-api                   |
+| ProductOfferServicePointAvailability                           | vendor/spryker/product-offer-service-point-availability                              |
+| ProductOfferServicePointAvailabilityCalculatorStorage          | vendor/spryker/product-offer-service-point-availability-calculator-storage           |
+| ProductOfferServicePointAvailabilityCalculatorStorageExtension | vendor/spryker/product-offer-service-point-availability-calculator-storage-extension |
+| ProductOfferServicePointAvailabilityStorage                    | vendor/spryker/product-offer-service-point-availability-storage                      |
+| ProductOfferServicePointAvailabilityStorageExtension           | vendor/spryker/product-offer-service-point-availability-storage-extension            |
+
+## 2. Extension point for calculating product offer service availability
```

Comment:
* Extension point for calculating the availability of a product offer at a service point.